server/app/services/customer_db_service.py

```
@@ -0,0 +1,472 @@
+            logger.info(f"Incremented call attempts to {customer.call_attempts} for customer {phone_number}")
+            return True
+            
+        except Exception as e:
+            logger.error(f"Error incrementing call attempts for phone {phone_number}: {e}")
+            return False
+    
+    @staticmethod
+    async def get_customers_ready_for_sms(
+        db: AsyncSession,
+        organization_id: str,
+        minutes_before_call: int = 20,
```

Ruff (0.14.8)

Unused static method argument: `minutes_before_call`

(ARG004)